doc/userguide/upgrade.rst

```diff
@@ -59,6 +59,8 @@ Other changes
 - NSS is no longer required. File hashing and JA3 can now be used without the NSS compile time dependency.
 - If installing Suricata without the bundled Suricata-Update, the ``default-rule-path`` has been changed from ``/etc/suricata/rules`` to ``/var/lib/suricata/rules`` to be consistent with Suricata when installed with Suricata-Update.
 - FTP has been updated with a maximum command request and response line length of 4096 bytes. To change the default see :ref:`suricata-yaml-configure-ftp`.
+- SWF decompression in http has been disabled by default. To change the default see :ref:`suricata-yaml-configure-libhtp`.
+  See https://redmine.openinfosecfoundation.org/issues/5632 for more information.
```

Comment:
Should we recommend that users upgrading to go and disable this?  As they won't pick up the new default behaviour.

Reply:
Should we actually recommend users with an old config where it may be enabled, to disable it? I think this makes some sense. It is probably useless to most setups now.

Reply:
> Should we actually recommend users with an old config where it may be enabled, to disable it?

With which medium ? forum.suricata.io  ?

Reply:
> > Should we actually recommend users with an old config where it may be enabled, to disable it?
> 
> With which medium ? forum.suricata.io ?

Upgrade docs I would think.

Reply:
Yeah, in the upgrade section of the docs.  If I'm upgrading from 6.0.x, I'll have this in my configuration file:
```
           swf-decompression:
             enabled: yes
             type: both
             compress-depth: 100kb
             decompress-depth: 100kb
```

Reply:
No problem :)